backends/bmv2/common/JsonObjects.h

```diff
@@ -49,7 +49,8 @@ class JsonObjects {
     void add_parser_transition(const unsigned id, Util::IJson* transition);
     void add_parser_op(const unsigned id, Util::IJson* op);
     void add_parser_transition_key(const unsigned id, Util::IJson* key);
-    void add_parse_vset(const cstring& name, const unsigned size);
+    void add_parse_vset(const cstring& name, const unsigned bitwidth,
+                        const big_int size);
```

Comment:
I said it was pointless to use `const unsigned&`, not that you shouldn't use `const big_int&`

Reply:
What is the reason that `const unsigned` is used without reference in the code base?  Using a reference with unsigned doesn't cause any damage. 
  
Then `const big_int` is used with reference because `big_int` is a typedef?

Reply:
There is a cost difference in copying an unsigned int vs. copying a big_int.

Actually passing the unsigned int by `const` reference can increase complexity and negatively impact performance: https://stackoverflow.com/a/3009568